api/metrics-api-jaxrs/src/main/java/org/hawkular/metrics/api/jaxrs/MetricHandler.java

```diff
@@ -847,7 +75,7 @@ public void onFailure(Throwable t) {
     public void findMetrics(
             @Suspended final AsyncResponse asyncResponse,
             @PathParam("tenantId") final String tenantId,
-        @ApiParam(value = "Queried metric type", required = true, allowableValues = "[num, avail, log]")
+        @ApiParam(value = "Queried metric type", required = true, allowableValues = "[gauge, avail, log]")
```

Comment:
From a UI perspective (any client really), I just want to add parameters to get the things I need and not have to switch urls (and maintain). 
So all I want is:
 /{tenantId}/metrics -> returns all metrics, regardless of the type
with http param filters.